### PR TITLE
Update arrays.dd

### PR DESCRIPTION
**Pull request by MrcSnm**
Mention new expression for initializing dynamic array length

### CODE REVIEW

**Review by dukc — CHANGES_REQUESTED** [review]
I think this one better befits the "Array Initialization" section rather than the "Array properties" section.

**Comment by MrcSnm** [comment]
> I think this one better befits the "Array Initialization" section rather than the "Array properties" section.

That section is actually `Setting Dynamic Array Length`. I didn't made into Array Initialization section because I saw it was only mentioning initializing it with data, and not its size.

**Comment by maxhaton** [comment]
```
Searching for trailing whitespace
./spec/arrays.dd:782:        
./spec/arrays.dd:784:        
./spec/arrays.dd:789:        
./spec/arrays.dd:796:
```

Whitespace checker says jump, we have to say "how high?".

**Comment by MrcSnm** [comment]
Did my best to edit it correctly, but idk how it really should be. Writing Ddoc is too strange for me yet

File: spec/arrays.dd [hunk 1]
```diff
@@ -780,6 +780,20 @@ $(H3 $(LNAME2 default-initialization, Default Initialization))
         $(LI Associative arrays are initialized to having 0 elements.)
         )
 
+$(H3 $(LNAME2 length-initialization, Length Initialization))
+
```
Comment (maxhaton) [inline spec/arrays.dd:784]:
```suggestion

```
Little bit of trailing whitespace here

**Review by dukc — APPROVED** [review]

**Review by maxhaton — APPROVED** [review]

**Pull request merged**